external-crates/move/move-compiler/src/expansion/ast.rs

```diff
@@ -230,7 +230,7 @@ pub enum Visibility {
 #[derive(PartialEq, Clone, Debug)]
 pub struct FunctionSignature {
     pub type_parameters: Vec<(Name, AbilitySet)>,
-    pub parameters: Vec<(Var, Type)>,
+    pub parameters: Vec<(Mutability, Var, Type)>,
```

Comment:
It might be worth making this into a struct now, a a three-tuple can be a bit vague.

Reply:
Eh, either way. I'm not too picky.